tests/jerry-test-suite/15/15.03/15.03.03/15.03.03.02/15.03.03.02-001.js

```diff
@@ -1,19 +0,0 @@
-// Copyright JS Foundation and other contributors, http://js.foundation
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-var desc = Object.getOwnPropertyDescriptor(Function, "length");
-assert(desc.value === 1 &&
-        desc.writable === false &&
-        desc.enumerable === false &&
-        desc.configurable === false);
```

Comment:
Perhaps we could move this test to ES5.1.

Reply:
During the exeuction of `jerry-test-suite` test, It seems that `es2015-subset` profile runs the whole test files under tests/jerry-test-suite directory, so I couldn't move this file.
Would you please give me a hint?
Or can I fix the test script to run files under `tests/jerry-test-suite/es2015` only for `es2015-subset` profile? (es5.1 profile runs files which are denoted in es51-profile-list, but there is no es2015-profile-list)

Reply:
You can move this test to here:
https://github.com/jerryscript-project/jerryscript/tree/master/tests/jerry/es5.1

Reply:
This test file is moved under tests/jerry/es5.1 directory.